### ISSUE TITLE
Improve VoiceOver support

### ISSUE THREAD

**Issue by vIiRuS** [HabitRPG/habitica-ios#326]
Currently the VoiceOver support is very limited and lacking. To be inclusive and accessible, we should improve that.

**Comment by bdorer** [comment]
May you want to split this into the following todos:
include + and - buttons into the VoiceOver list. At the moment I'm able to edit and delete created todos etc.
make notification windows and its buttons focusable.
Lable symbols like gems, gold and so on.

If you need help testing VoiceOver support please feel free to ask.
